### PR TITLE
Finalize naming

### PR DESCRIPTION
**Pull request by ruebot**
**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/592

# What does this Pull Request do?

Last of clean-up after changing the repo name.

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-    "name": "islandora/claw-jsonld",
+    "name": "islandora/jsonld",
```
Comment (DiegoPino) [inline composer.json]: if name changes then @dannylamb needs to update https://github.com/Islandora-CLAW/islandora/pull/53/files/64e4386bfa1c5e98b9b63cf37534a5bf600c4843..9686c9054d3fe8c3fce54b4414e794b37035f7f5#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R21. Also other modules (not sure which ones) need their composer updated if they depend on this package

**Comment by whikloj** [comment]
Also to follow with what Diego is saying about updating composer.json in Islandora. We will need to immediately cut a release as this would be considered a new project on packagist.

**Comment by dannylamb** [comment]
Looks like packagist sees no distinction between claw-jsonld and jsonld.  If we want to rename, we'll have to kill claw-jsonld for good on packagist.

**Comment by dannylamb** [comment]
Looks like I'll have to get in touch with support to get rid of it for good.  This'll have to sit until then.

**Comment by dannylamb** [comment]
k, i'm gonna have to update some stuff on packagist after this is merged before the rename'll take hold.  are we ready for this?

**Pull request merged**